samples/tapi/hog.cpp

```diff
@@ -76,10 +76,8 @@ int main(int argc, char** argv)
         "{ s scale     | 1.0            | resize the image before detect}"
         "{ o output    |                | specify output path when input is images}";
     CommandLineParser cmd(argc, argv, keys);
-    if (cmd.has("help"))
```

Comment:
@dozyc I believe there is misunderstanding here. @paroj means to remove only one default value for "help", not all default values.